.github/workflows/deploy.yml

```diff
@@ -0,0 +1,32 @@
+name: Deploy
```

Comment:
one small question - why not just have a singular `definitions.yaml` file which contains both the contents of `build-and-test.yml` and `deploy.yml`. Is it just for the sake of clarity or is there a bigger reason I'm missing 🤔

Reply:
The triggers and environments associated with these workflows are different, so I think it's best to keep them separate. Deploy only happens on a release, while build and test happens on every push.